### ISSUE TITLE
Wrong var name for debug file

### ISSUE THREAD

**Issue by BenoitPoulet** [NagiosEnterprises/nrdp#37]
Hello,

The var name for debug file is wrong in the "config.inc.php" :

```
// Where should the logs go?
$cfg["debug_log"] = "/home/nrdp/server/debug.log";
```

But in 
"server/includes/utils.inc.php" , line 406
You are using another name for the variable

```
$file = grab_array_var($cfg, "debug_file", "/usr/local/nrdp/server/debug.log");
```

**Comment by hedenface** [comment]
The default installation is `/usr/local/nrdp` - if you change this to `/home/nrdp/` then you should also change that variable's value.

**Comment by hedenface** [comment]
How did you install nrdp?

**Comment by BenoitPoulet** [comment]
by hands, git clone.

That't not the variable value the pb, that is the variable **name**

debug_log <> debug_file

**Comment by hedenface** [comment]
Oooooh I see. Thanks. Will update.

**Comment by jomann09** [comment]
This was fixed with merge request #36
